packages/abi-coder/src/abi-coder.ts

```diff
@@ -16,18 +16,60 @@ import StringCoder from './coders/string';
 import StructCoder from './coders/struct';
 import TupleCoder from './coders/tuple';
 import U64Coder from './coders/u64';
+import VecCoder from './coders/vec';
```

Comment:
We should also include `export *  from ./coders/vec` on the abi-coder index.ts file